src/Symfony/Component/DependencyInjection/Compiler/RegisterConstructorPass.php

```diff
@@ -24,9 +24,12 @@ final class RegisterConstructorPass implements CompilerPassInterface
     public function process(ContainerBuilder $container): void
     {
         foreach ($container->findTaggedServiceIds('container.from_constructor') as $id => $tags) {
-            if (1 !== count($tags) || !key_exists('method', $tags[0])) {
+            if (1 !== count($tags)) {
                 throw new AutoconfigureFailedException($id, sprintf('Multiple tags "container.from_constructor" attached to service %s. The #[Constructor] attribute can only be applied to one method.', $id));
             }
+            if (!key_exists('method', $tags[0])) {
```

Comment:
https://www.php.net/manual/en/function.array-key-exists.php because `key_exists` is just an alias prefer the original one.